api/desecapi/throttling.py

```diff
@@ -0,0 +1,58 @@
+from rest_framework import throttling
+from rest_framework.settings import api_settings
+
+
+class ScopedRatesThrottle(throttling.ScopedRateThrottle):
+    """
+    Like DRF's ScopedRateThrottle, but supports several rates per scope, e.g. for burst vs. sustained limit.
+    """
+    def parse_rate(self, rates):
+        return [super(ScopedRatesThrottle, self).parse_rate(rate) for rate in rates]
```

Comment:
I think just `super().parse_rate` does the job here

Reply:
[nope](https://stackoverflow.com/a/31895448/6867099), as `self` is undefined within list comprehensions (it also did not make sense to me intuitively)